app/src/main/java/com/asfoundation/wallet/topup/payment/AdyenTopUpPresenter.kt

```diff
@@ -224,7 +224,12 @@ class AdyenTopUpPresenter(private val view: AdyenTopUpView,
         .observeOn(networkScheduler)
         .flatMapSingle { adyenPaymentInteractor.disablePayments() }
         .observeOn(viewScheduler)
-        .doOnNext { success -> if (!success) handleSpecificError(R.string.unknown_error) }
+        .doOnNext { success ->
+          if (!success) {
+            logger.log(TAG, "Unabled to forget card")
+            handleSpecificError(R.string.unknown_error)
+          }
```

Comment:
Maybe we should move the logger to handleSpecificError in order to remove boilerplate code